src/main/java/no/uib/inf112/core/map/tile/Attribute.java

```diff
@@ -52,6 +52,21 @@
      * Mark this movable tile to be pushed if something moves onto it
      */
     PUSHABLE(MovableTile.class),
+
+    /**
+     * This tile shoots a laser
+     */
+    SHOOTS_LASER,
+
+    /**
+     * This tile lays down laser, meaning it puts down laser on the entitylaser layer
+     * e.g robots
+     */
+    LAYS_DOWN_LASER,
```

Comment:
I'm confused on what this does. Does it change the layer a laser will be on?

Reply:
The reason for this is that it requires different logic with a robot laser than a wall laser, therefore lays down laser is a laser that creates laser tiles in the entitylaser, but i can clarify this in the JavaDoc